bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -1,195 +1,432 @@
-Switch CentralHeating2EnabledOverride "Central 2 heating enabled override" <switch> { channel="openthermgateway:otgw:1:ch2_enableoverride" }
-Switch CentralHeatingActive "Central heating active" <switch> { channel="openthermgateway:otgw:1:ch_mode" }
-Switch DomesticHotWaterEnabled "Domestic hot water enabled" <switch> { channel="openthermgateway:otgw:1:dhw_enable" }
-Switch DomesticHotWaterActive "Domestic hot water active" <switch> { channel="openthermgateway:otgw:1:dhw_mode" }
-Switch BurnerActive "Burner active" <switch> { channel="openthermgateway:otgw:1:flame" }
-Number:Dimensionless RelativeModulationLevel "Relative modulation level [%.1f %%]" { channel="openthermgateway:otgw:1:modulevel" }
-Number:Dimensionless MaximumRelativeModulationLevel "Maximum relative modulation level [%.1f %%]" { channel="openthermgateway:otgw:1:maxrelmdulevel" }
-Switch Fault "Fault indication" <switch> { channel="openthermgateway:otgw:1:fault" }
-Switch ServiceRequest "Service required" <switch> { channel="openthermgateway:otgw:1:servicerequest" }
-Switch LockoutReset "Lockout-reset" <switch> { channel="openthermgateway:otgw:1:lockout-reset" }
-Switch LowWaterPress "Low water pressure fault" <switch> { channel="openthermgateway:otgw:1:lowwaterpress" }
-Switch GasFlameFault "Gas or flame fault" <switch> { channel="openthermgateway:otgw:1:gasflamefault" }
-Switch AirPressFault "Air pressure fault" <switch> { channel="openthermgateway:otgw:1:airpressfault" }
-Switch WaterOvTemp "Water over-temperature fault" <switch> { channel="openthermgateway:otgw:1:waterovtemp" }
-Number OemFaultCode "OEM fault code" { channel="openthermgateway:otgw:1:oemfaultcode" }
-Switch Diagnostics "Diagnostics indication" { channel="openthermgateway:otgw:1:diag" }
-Number:Dimensionless UnsuccessfulBurnerStarts "Unsuccessful burner starts" { channel="openthermgateway:otgw:1:unsuccessfulburnerstarts" }
-Number:Dimensionless BurnerStarts "Burner starts" { channel="openthermgateway:otgw:1:burnerstarts" }
-Number:Dimensionless CentralHeatingPumpStarts "Central heating pump starts" { channel="openthermgateway:otgw:1:chpumpstarts" }
-Number:Dimensionless DomesticHotWaterPumpValveStarts "Domestic hot water pump/valve starts" { channel="openthermgateway:otgw:1:dhwpvstarts" }
-Number:Dimensionless DomesticHotWaterBurnerStarts "Domestic hot water burner starts" { channel="openthermgateway:otgw:1:dhwburnerstarts" }
-Number:Time BurnerHours "Burner hours" { channel="openthermgateway:otgw:1:burnerhours" }
-Number:Time CentralHeatingPumpHours "Central heating pump hours" { channel="openthermgateway:otgw:1:chpumphours" }
-Number:Time DomesticHotWaterPumpValveHours "Domestic hot water pump/valve hours" { channel="openthermgateway:otgw:1:dhwpvhours" }
-Number:Time DomesticHotWaterBurnerHours "Domestic hot water burner hours" { channel="openthermgateway:otgw:1:dhwburnerhours" }
-Text SendCommand "Send command channel" { channel="openthermgateway:otgw:1:sendcommand" }
-```
-
-### demo.sitemap
+#### Example
```

Comment:
To be consistent with the other binding documentations, the examples should reside at the end of the readme.

Reply:
I have struggled a bit with this. There are multiple things  and I could either document each thing directly followed by an example, or split it up and first document all configs of all things, then all channels of all things, followed by all examples.  Which to me makes it scattered around and hard to read.

Regarding consistency... I haven't really found any :) especially when it comes to bindings with bridges and multiple things.

Reply:
Done